spec/repositories/manual_repository_spec.rb

```diff
@@ -163,20 +161,18 @@
         .with(arguments)
     end
 
-    it "returns the built manual" do
-      expect(repo[manual_id]).to be(manual)
-    end
-
```

Comment:
I assume the justification for removing this is:

> The `it returns the built manual` spec is now double-tested so it has also been removed.

However, I don't completely understand what you mean by "double-tested". Can you elaborate?